libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/InitialGpsEvent.kt

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import android.annotation.SuppressLint
+import android.os.Parcelable
+
+@SuppressLint("ParcelCreator")
+internal data class InitialGpsEvent(
+    private val elapsedTime: Double,
+    private val sessionId: String,
+    private val metadata: NavigationPerformanceMetadata?
```

Comment:
Couldn't be this non-nullable? 👀 https://github.com/mapbox/mapbox-navigation-android/blob/06f447459bc2553f754c4c74d6be44e739bf37d0/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/MetadataBuilder.kt#L29 https://github.com/mapbox/mapbox-navigation-android/blob/06f447459bc2553f754c4c74d6be44e739bf37d0/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationTelemetry.java#L136